conda/recipes/libcugraph/meta.yaml

```diff
@@ -28,12 +28,10 @@ requirements:
     - libcudf=0.8*
     - cython
     - cudatoolkit {{ cuda_version }}.*
-    - boost
```

Comment:
Do we still need it for build to ensure the needed headers are present, or are you relying on boost being installed at the system level?

Reply:
my local build seems to get boost's headers from the conda env

```
/cpp/build/CMakeFiles/cugraph.dir/depend.internal:18: /home/afender/anaconda3/envs/cugraph_dev/include/boost/config.hpp
...
```